res/travis/travis_build.expect

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/expect -f
+set timeout 300
+set prompt "(%|#|\\$) $"
+
+eval spawn $argv
+
+expect {
+    "Please acknowledge these tool dependencies. Do you wish to proceed? " {
+      send -- "Y"
+      exp_continue
+    }
+
+    "Do you agree to the license of this third party library? (Y) " {
+      send -- "Y"
+      exp_continue
+  }
+}
```

Comment:
congrats to the new "expect" "technology" added to the project, great job! Thank you for showing me this.